ding/entry/serial_entry_sqil.py

```diff
@@ -59,6 +59,7 @@ def serial_pipeline_sqil(
     expert_cfg = compile_config(
         expert_cfg, seed=seed, env=env_fn, auto=True, create_cfg=expert_create_cfg, save_cfg=True
     )
+    expert_cfg.policy.collect.n_sample = cfg.policy.collect.n_sample
```

Comment:
why add this line? add some comments

Reply:
The purpose of adding this line is to fix the bug that if n_sample in sqil is different from that in the expert config, the code will report a bug in forward learn phase